node.toml.erb

```diff
@@ -47,7 +46,7 @@ HTTPHost = "0.0.0.0"
 HTTPPort = <%= @Node_HTTPPort %>
 HTTPCors = ["*"]
 HTTPVirtualHosts = ["*"]
-HTTPModules = ["db", "eth", "net", "web3", "personal", "web3"]
+HTTPModules = ["db", "eth", "net", "web3", "personal", "miner"]
```

Comment:
is 'miner' really a module? i can see `web3` is redundant, so maybe we just drop it from the list?